control_panel_api/settings.py

```diff
@@ -122,3 +122,8 @@
 
 LOGIN_URL = 'rest_framework:login'
 LOGOUT_URL = 'rest_framework:logout'
+
+# AWS variables
+BUCKET_REGION = os.environ.get('BUCKET_REGION', 'eu-west-1')
+ENV = os.environ.get('ENV', 'dev')
```

Comment:
This is necessarily an AWS-related setting.
Also, we'll need to find a way to set these environment variables when deploying (didn't look at that part)

Reply:
That will all be devops side e.g. Andy

Reply:
Yeah, I was adding the role to the helm chart and as I was in there I added these environment variables you need ;)